addons/Image_Viewer_Public/src/presenter.js

```diff
@@ -1413,10 +1413,12 @@ function AddonImage_Viewer_Public_create() {
             presenter.triggerFrameChangeEvent(presenter.configuration.currentFrame + 1);
         }
     };
-    presenter.changeFrame = function(isPreview, isReverseOrder, triggerEvent) {
+    presenter.changeFrame = function(isPreview, isReverseOrder, triggerEvent, readFrame) {

```

Comment:
"Drobnostka": można by parametry zmienić w obiekt, bo czytanie wywołania 
`changeFrame(false, true,false, true)` 
totalnie nic nie mówi o parametrach, za to:
```
changeFrame({
     isPreview: false,
     isReverseOrder: true,
     triggerEvent: false,
     readFrame: true
})
```
już tak.